File: 3783/3783.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var mirrorDistanceOfAnInteger = function(root) {
    if (!root) return 0;
    return 1 + Math.max(mirrorDistanceOfAnInteger(root.left), mirrorDistanceOfAnInteger(root.right));
};

File: 3783/3783.go
func mirrorDistanceOfAnInteger(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := mirrorDistanceOfAnInteger(root.Left)
	r := mirrorDistanceOfAnInteger(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}